### ISSUE TITLE
frozen

### ISSUE THREAD

**Issue by GrayJediMaster** [stackblitz/bolt.new#1951]
### Describe the bug

{
  "name": "dalle-image-generator",
  "private": true,
  "version": "1.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "openai": "^4.20.1",
    "dotenv": "^16.3.1"
  },
  "devDependencies": {
    "vite": "^5.0.0"
  }
}

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-uulmrm

### Steps to reproduce

dall-e 3 api deploy ment failed to netlify

### Expected behavior

??

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 56855504,
  "usedJSHeapSize": 52390168,
  "jsHeapSizeLimit": 4294705152
}
Username = GrayJediMaster
Chat ID = 1c417c98fff7
```

**Comment by endocytosis** [comment]
Hi @GrayJediMaster, 

Are you saying the app failed to deploy to Netlify when you clicked the Deploy button. Did you receive a specific error message?

**Comment by GrayJediMaster** [comment]
Yes! I would've spent maybe five hours trying to get it to deploy and when
it did there were initially issues because even though I had put the API
key in the .ENV section in Netlify - it was not able to retrieve it.

Then we tried to deploy it on a different platform by modifying front and
back codes and adding api in wix secret  - which also failed.

Then we went back to Netlify and when it did publish and I have the API key
in Netlify, it did not Deploy properly.
Meaning the preview inside of Molt worked but inside of Netlify it didn't.
I also noticed that when I tried to restore to prior versions that were
working - they didn't work any more.
I also downloaded the entire Project stack of code onto my hard drive and
uploaded back because those are the working versions and it still didn't
work when I did.

It seems to edit the code randomly even when not prompted, same way that if
you use ChatGPT or Claude they do hallucinate so maybe there's a
hallucination issues when it restores. There are other issues but I can't
remember.

I got many project files and can share them if need be.

Dany
